README.md

```
@@ -98,6 +98,119 @@ Add the following snippet to the `<head>` section of the webpage where you want
 </script>
 ```
 
+## API Usage
+The chatbot exposes a REST API that allows developers to integrate chatbot functionality into their applications.
+
+### Endpoint
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)